Cargo.toml

```diff
@@ -52,6 +53,11 @@ walkdir = "2"
 async-recursion = "1.0.5"
 thiserror = "1.0.49"
 miette = { version = "5.10.0", features = ["fancy"] }
+protobuf = { version = "3.3.0", optional = true }
+protobuf-parse = { version = "3.3.0", optional = true }
+diff-struct = { version = "0.5.3", optional = true }
+derive_more = "0.99.17"
+anyhow = "1.0.75"
```

Comment:
```suggestion
```

Please move to `miette`

Reply:
I am using `miette`. This is only needed because I want to capture an error returned by `protobuf_parse`, who use that as their error type.